README.md

```diff
@@ -6,7 +6,7 @@ HackMD
 [![Join the chat at https://gitter.im/hackmdio/hackmd][gitter-image]][gitter-url]
 [![build status][travis-image]][travis-url]
 [![version][github-version-badge]][github-release-page]
-
+[![Help Contribute to Open Source](https://www.codetriage.com/hackmdio/hackmd/badges/users.svg)](https://www.codetriage.com/hackmdio/hackmd)
```

Comment:
Could you use the reference schema like the other links? keeps the Markdown a bit more readable 😄